src/app/locator/qgsinbuiltlocatorfilters.h

```diff
@@ -113,6 +113,41 @@ class QgsActiveLayerFeaturesLocatorFilter : public QgsLocatorFilter
     QIcon mLayerIcon;
 };
 
+class QgsAllLayersFeaturesLocatorFilter : public QgsLocatorFilter
+{
+    Q_OBJECT
+
+  public:
+    struct PreparedLayer
+    {
+      public:
+        QgsExpression expression;
+        QgsExpressionContext context;
+        QgsFeatureIterator iterator;
+        QString layerName;
+        QString layerId;
+        QIcon layerIcon;
+    } ;
+
+    QgsAllLayersFeaturesLocatorFilter( QObject *parent = nullptr );
+    QgsAllLayersFeaturesLocatorFilter *clone() const override;
+    QString name() const override { return QStringLiteral( "allfeatures" ); }
+    QString displayName() const override { return tr( "Features In All Layers" ); }
+    Priority priority() const override { return Medium; }
+    QString prefix() const override { return QStringLiteral( "a" ); }
```

Comment:
Can you choose a less generic prefix here? Maybe "af" (for All Features?)

Reply:
done